From 27a37059d4eefc9852ed09d574bf9e87c6b6110e Mon Sep 17 00:00:00 2001
From: Ran Shalit <rshalit@habana.ai>
Date: Sun, 04 Aug 2024 18:51:19 +0300
Subject: [PATCH] [FSW-7946] fs1: ppboot: onix: remove cpu-id

This patch is temporarily removing cpu if from
fs1 onix, because it is not supported in fs1 Onix.

Change-Id: I5b8b3564a26cbedc48e58d61edcb57cc8fb54297
Signed-off-by: Ran Shalit <rshalit@habana.ai>
---

diff --git a/arch/fs1/arc/common.c b/arch/fs1/arc/common.c
index 84f3c42..8fd37c6 100644
--- a/arch/fs1/arc/common.c
+++ b/arch/fs1/arc/common.c
@@ -218,7 +218,7 @@
 			   mmARC_LOCAL_CONF_ARC_LBU_AXI_SPLIT_CTRL2);
 }
 
-static void hl_init_cpu_ids(void)
+__weak void hl_init_cpu_ids(void)
 {
 	regmap_writel(IO0_PSOC_ARC_RST_ID, mmIO_0_CFG_HALF_0_PSOC_ARC_RST_ARC_BASE + mmARC_ARC_NUM);
 	regmap_writel(IO0_PSOC_ARC_PID_ID, mmIO_0_CFG_HALF_0_PSOC_ARC_PID_ARC_BASE + mmARC_ARC_NUM);
diff --git a/arch/fs1/arc/onix.c b/arch/fs1/arc/onix.c
index 82ca3cc..4f16bfe 100644
--- a/arch/fs1/arc/onix.c
+++ b/arch/fs1/arc/onix.c
@@ -60,3 +60,6 @@
 
 /* Early write response not available in Onix */
 void hl_lbw_early_write_resp_enable(void) {}
+
+/* Init cpu-id not available in Onix */
+void hl_init_cpu_ids(void) {}
